apps/masterbots.ai/components/layout/profile/profile-sidebar.tsx

```
@@ -0,0 +1,83 @@
+      <div className="w-64 border-r bg-gray-50 dark:bg-black flex flex-col">
+        <nav className="flex-1 overflow-y-auto scrollbar">
+          <div className="flex flex-col space-y-1 mt-3 font-Geist">
+            {/* Threads Accordion */}
+            <div className="rounded-lg">
+              <button
+                onClick={() => setIsThreadsOpen(!isThreadsOpen)}
+                className={cn(
+                  "flex w-full items-center justify-between px-4 py-3 hover:bg-gray-200 dark:hover:bg-mirage",
+                  isThreadsOpen ? 'bg-gray-200 dark:bg-mirage' : ''
+                )}
+              >
```

Biome

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)